mlir/lib/Dialect/AMDGPU/Transforms/OptimizeSharedMemory.cpp

```diff
@@ -253,4 +278,8 @@ struct OptimizeSharedMemoryPass
         return;
     }
   }
+
+private:
+  int64_t _kSharedMemoryLineSizeBytes;
```

Comment:
Not sure if this leading underscore is LLVM style. But you shouldn't need it anyway if you get rid of the globals.